domain-models-runtime/src/test/java/org/folio/rest/persist/PgExceptionFacadeTest.java

```diff
@@ -24,10 +24,16 @@ void getter() {
   }
 
   @Test
-  void isInvalidTextRepresentation() {
+  void invalidTextRepresentation() {
     PgExceptionFacade f = new PgExceptionFacade(new PgException(null, null, "22P02", null));
     assertThat(f.isInvalidTextRepresentation(), is(true));
-    PgExceptionFacade f2 = new PgExceptionFacade(new PgException(null, null, "22P03", null));
-    assertThat(f2.isInvalidTextRepresentation(), is(false));
+    assertThat(f.isVersionConflict(), is(false));
+  }
+
+  @Test
+  void versionConflict() {
+    PgExceptionFacade f = new PgExceptionFacade(new PgException(null, null, "23F09", null));
```

Comment:
Might make it easier to follow if you used the constant `PgExceptionUtil.VERSION_CONFLICT` to set the 3rd parma here and elsewhere just to show that these strings are not just random but have meaning.